### DISCUSSION TITLE
Cell type options [Apr 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5547] [Discord Threads]
> 🙋‍♂️ **Hioko**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Relations - NocoDB supports has-many & many-to-many. You can find more details here. https://docs.nocodb.com/setup-and-usages/link-to-another-record  
#  
> 🙋‍♂️ **Hioko**  
#### thanks <@857998597474484295> , i ll look into it  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1087078865660293262)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
